chrome/content/zotero/preferences/preferences_general.xhtml

```diff
@@ -160,21 +160,21 @@
 		<groupbox>
 			<label><html:h2>&zotero.preferences.prefpane.locate;</html:h2></label>
 			
-			<hbox>
+			<hbox align="center">
+				<label control="openurl-menu" value="&zotero.preferences.openurl.choose;"></label>
```

Comment:
The string here is "Choose a resolver…", so we can't just stick that in front of a drop-down. We can add a new Fluent string for "Resolver" and change "Resolver:" to "Base URL" (taking the opportunity to remove the unnecessary colons).

And "Version:" I think we can just remove and hard-code, since 1.0 came out before Zotero even existed.